src/tidelift_client.ts

```diff
@@ -22,25 +22,23 @@ export class TideliftClient {
     /* eslint-enable @typescript-eslint/naming-convention */ //
   }
 
-  async fetch_recommendation(
+  async fetch_vulnerability(
     vuln: VulnerabilityId
-  ): Promise<TideliftRecommendation | undefined> {
-    const response = await this.client.get(
-      `/vulnerability/${vuln}/recommendation`
-    )
+  ): Promise<Vulnerability | undefined> {
+    const response = await this.client.get(`/vulnerabilities/${vuln}`)
 
     if (response.status === 404) {
       return
     }
 
-    return new TideliftRecommendation(vuln, response.data)
+    return new Vulnerability(vuln, response.data)
```

Comment:
Since we no longer 404 when we don't have a recommendation maybe we should filter out Vulnerabilities that have no recommendation in here or maybe from a different function that would be called before trying to format a message when we don't have any data to show.